File: FRONT/src/app/models/venda.ts
import { Categoria } from "./categoria";
import { FormaPagamento } from "./forma-pagamento";
import { ItemVenda } from "./item-venda";

export interface Venda {
    vendaId?: number;
    cliente: string;
    formaPagamentoId?: number;
    carrinhoId: string;
    criadoem?: Date;
}

File: API/Models/Venda.cs
using System;
using System.Collections.Generic;

namespace API.Models
{
    public class Venda
    {
        public Venda() => CriadoEm = DateTime.Now;
        public int VendaId { get; set; }
        public string Cliente { get; set; }
        public FormaPagamento FormaPagamento { get; set; }
        public int FormaPagamentoId { get; set; }
        public string CarrinhoId { get; set; }
        public List<ItemVenda> Itens { get; set; }
        public DateTime CriadoEm { get; set; }
    }
}